DigitalLearningSolutions.Web/Views/MyAccount/EditDetails.cshtml

```diff
@@ -15,7 +15,7 @@
 <div class="nhsuk-grid-row">
   <div class="nhsuk-grid-column-full">
     @if (errorHasOccurred) {
-      <vc:error-summary order-of-property-names="@(new[] { nameof(Model.FirstName), nameof(Model.LastName), nameof(Model.Email), nameof(Model.Password) })" />
+      <vc:error-summary order-of-property-names="@(new[] { nameof(Model.FirstName), nameof(Model.LastName), nameof(Model.Email), nameof(Model.Password), nameof(Model.ProfessionalRegistrationNumber), nameof(Model.HasProfessionalRegistrationNumber) })" />

```

Comment:
I think the new properties are the wrong way round; we want `HasProfessionalRegistrationNumber` before `ProfessionalRegistrationNumber`

Reply:
Good spot, done.